src/views/Home.jsx

```diff
@@ -1,28 +1,64 @@
 import './Home.css';
+import { useState } from 'react';
 import { SingleList } from '../components/SingleList.jsx';
+import { createList } from '../api/index.js';
+import { auth } from '../api/config.js';
+import { useNavigate } from 'react-router-dom';
 
 export function Home({ data, setListPath }) {
-	const singleList = data.map((data) => {
-		return (
-			<SingleList
-				key={data.name}
-				name={data.name}
-				path={data.path}
-				setListPath={setListPath}
-			/>
-		);
-	});
+	const navigate = useNavigate();
+	const [listName, setListName] = useState('');
+	const [message, setMessage] = useState('');
+
+	const handleChange = (e) => {
+		setListName(e.target.value);
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		try {
+			await createList(auth.currentUser.uid, auth.currentUser.email, listName);
+			setMessage('List created');
```

Comment:
I mentioned this in our call add a comma like `List created, redirecting in 1 second...`